apis/Google.Cloud.PubSub.V1/docs/index.md

```diff
@@ -2,6 +2,17 @@
 
 {{description}}
 
+# Class renaming in [v1.0.0-beta16](https://www.nuget.org/packages/Google.Cloud.PubSub.V1/1.0.0-beta16)
+
+The following classes have been renamed when moving from version 1.0.0-beta15 to 1.0.0-beta16:
+
+* `SimplePublisher` is now named [PublisherClient](obj/api/Google.Cloud.PubSub.V1.PublisherClient.yml)
```

Comment:
Aren't there settings classes for these as well?

Reply:
These settings classes are nested, so they haven't directly changed.